project/plugins.sbt

```diff
@@ -6,8 +6,6 @@ addSbtPlugin("org.coursera.courier" % "courier-sbt-plugin" % "2.1.4")
 
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
I removed this because this dependency is so old that it was removed from Maven. There's a newer version at `com.github.sbt:sbt-pgp:2.1.2`, but it doesn't seem to be compatible with sbt 0.13.

I *think* this still works because it seems sbt 0.13 actually bundles a copy of the original `com.jsuereth` jar.